src/inmanta/execute/runtime.py

```diff
@@ -337,13 +341,59 @@ def set_value(self, value: ListValue, location: Location, recur: bool = True) ->
             # any set_value may fulfill a promise, allowing this object to be queued
             if self.can_get():
```

Comment:
Shouldn't this just be return True? The same on line 330?

Reply:
no further processing is required here, as the list was not changed, same for 330.